images/icons/zeppelin_svg_logo.svg

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Is it okay to use this asset?

Reply:
Yeah I think so? 
I downloaded from the official website https://zeppelin.apache.org/assets.html.